components/helm-broker/internal/helm/client_test.go

```diff
@@ -2,25 +2,74 @@ package helm_test
 
 import (
 	"context"
+	"encoding/pem"
+	"fmt"
+	"io/ioutil"
 	"net"
+	"os"
 	"testing"
 	"time"
 
+	"github.com/SpectoLabs/hoverfly/core/certs"
 	"github.com/ghodss/yaml"
 	"github.com/kyma-project/kyma/components/helm-broker/internal"
 	"github.com/kyma-project/kyma/components/helm-broker/internal/helm"
 	"github.com/kyma-project/kyma/components/helm-broker/platform/logger/spy"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials"
 	"k8s.io/helm/pkg/proto/hapi/chart"
 	hapi_release5 "k8s.io/helm/pkg/proto/hapi/release"
 	"k8s.io/helm/pkg/proto/hapi/services"
 )
 
+var (
+	keyFile  string
+	certFile string
+)
+
+func TestMain(m *testing.M) {
+	setupCerts()
+	code := m.Run()
+	cleanupCerts()
+	os.Exit(code)
+}
+
+func setupCerts(){
+	certOut, err := ioutil.TempFile("/tmp/", "certFile")
+	if err != nil {
+		fmt.Println(err)
```

Comment:
probably you can replace it with log.Fatal

Reply:
done